<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIsearchDialog</class>
 <widget class="QDialog" name="UIsearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>136</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="searchway">
         <property name="maximumSize">
          <size>
           <width>171</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>按学号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按姓名</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按专业班级</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="useMajor">
         <property name="maximumSize">
          <size>
           <width>171</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="useClass">
         <property name="maximumSize">
          <size>
           <width>171</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="idName">
         <property name="maximumSize">
          <size>
           <width>171</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>136</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>UIsearchDialog</receiver>
   <slot>Accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>UIsearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchway</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>UIsearchDialog</receiver>
   <slot>waychange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useMajor</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>UIsearchDialog</receiver>
   <slot>majorChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useClass</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>UIsearchDialog</receiver>
   <slot>classChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idName</sender>
   <signal>editingFinished()</signal>
   <receiver>UIsearchDialog</receiver>
   <slot>endedit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>waychange(int)</slot>
  <slot>majorChange(int)</slot>
  <slot>classChange(int)</slot>
  <slot>endedit()</slot>
  <slot>Accept()</slot>
 </slots>
</ui>
